### ISSUE TITLE
Let Users Name, Describe, and Suppress topics

### ISSUE THREAD

**Issue by tjowens** [edsu/fondz#17]
It would be neat if an archivist using the tool could have ways to add value and make the results more useful to potential end users (researchers, etc.). So, I imagine giving the archivist the ability to put a name on a topic and a short sentence or two about what that topic is about. That is, they can click into the topic and use their judgement to give a name, and potentially a few sentences of description for the topic. Along with that, if they see topics that don't really seem to hang together, it would be nice have a way for them to suppress a topic. That is, put in a note that results in the topic just not being displayed. 

**Where to put this in the workflow**
I could imagine this being activated directly off the topics list page, like a button that says "Name" and a button that says "Suppress" next to each topic. So you click the button to "Name" a topic and then you see a one line text box to put in the name and a 3 or 4 line text box where the user can type in a short set of sentences explaining the name. 

**Where to store Names, Descriptions and Suppression flags**
My first thought is that the best way to do this would be to have all this text written to another json file or something. So that the tool isn't messing with the reports and results of the automated processes but is instead adding a separate set of information that provides data that comes together in the display.

**Comment by edsu** [comment]
The trick is that right now, there is no web application per-se, just a python program (fondz) that reads a bunch of bags, and emits a bunch of static HTML, CSS, JavaScript and JSON. The idea being that fondz (or one of its many dependencies) could break or become unavailable, and the description and content would still be readable assuming Web browsers are still around. 

So having a button on the Web page that changes the underlying HTML/JSON may be problematic, but I think it's worth looking into what browsers can do these days. Perhaps there could be a browser extension that let you modify the fondz description too. Or if they aren't afraid of the command line, a sub-command could be added to fondz for naming/describing topic groups.

I guess I was thinking that archivists could edit the HTML themselves, if it was clear where and how to do it. Adding a sidecar JSON file would work well too I think. Thanks for entering an issue ticket for this idea, it has been rattling around in my brain too.

**Comment by tjowens** [comment]
Good point on just editing the HTML or JSON or something. So, for instance, if there was a file that was like name_describe_supress.json that could just have a place where the archivist could go in and fill in this information or something.
